docs/guides/vault_secrets_manager.md

```diff
@@ -8,15 +8,7 @@ description: |-
 
 ### Overview
 
-This guide outlines the process of utilizing the HashiCorp Vault provider alongside the STACKIT provider to write secrets into the STACKIT Secrets Manager. Due to the current limitation in the STACKIT Terraform provider where direct writing of secrets isn't supported, this workaround is necessary.
-
-### Problem
-
-Writing credentials from provisioned STACKIT services into a Secrets Manager instance is currently not supported by the STACKIT Terraform provider.
-
-### Solution
-
-To overcome this limitation, you can integrate the HashiCorp Vault provider with the STACKIT provider. This solution involves configuring the Vault provider to interface with the STACKIT Secrets Manager.
+This guide outlines the process of utilizing the HashiCorp Vault provider alongside the STACKIT provider to write datasources from STACKIT cloud resources (deployed with Terraform) as secrets in the STACKIT Secrets Manager.
```

Comment:
It's not restricted to data from STACKIT resources. Any secret can be written into the instance

Reply:
I updated the wording, I think it still makes sense to focus on stackit resources, since I think this will be the main usage of the guide, otherwise its just a guide for the Vault provider. Does this make sense?

Reply:
Yeah that makes sense :+1: